Koku UI yarn build failure - "Cannot find module 'jest-emotion'" && "Property 'toHaveStyleRule' does not exist on type 'Matchers<ShallowWrapper<any, Readonly<{}>, Component<{}, {}, any>>>'."

**Describe the bug**
When building Koku UI, it fails during the yarn build with the error below

```
ERROR in /opt/koku-ui/src/components/textInput/textInput.test.tsx
[tsl] ERROR in /opt/koku-ui/src/components/textInput/textInput.test.tsx(3,32)
      TS2307: Cannot find module 'jest-emotion'.

ERROR in /opt/koku-ui/src/components/textInput/textInput.test.tsx
[tsl] ERROR in /opt/koku-ui/src/components/textInput/textInput.test.tsx(32,5)
      TS2339: Property 'toHaveStyleRule' does not exist on type 'Matchers<ShallowWrapper<any, Readonly<{}>, Component<{}, {}, any>>>'.                                                                                                       

ERROR in /opt/koku-ui/src/components/textInput/textInput.test.tsx
[tsl] ERROR in /opt/koku-ui/src/components/textInput/textInput.test.tsx(38,5)
      TS2339: Property 'toHaveStyleRule' does not exist on type 'Matchers<ShallowWrapper<any, Readonly<{}>, Component<{}, {}, any>>>'.                                                                                                       
Child html-webpack-plugin for "index.html":
     1 asset
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
The command '/bin/sh -c yarn --production --non-interactive     && yarn build' returned a non-zero code: 1

```

**To Reproduce**
Steps to reproduce the behavior:
1. I modified the DockerFile to remove the yarn build output redirection to /dev/null 
2. Run "docker build" against the modified Dockerfile to see the error
The OCP build also fails with 


```
Step 9/13 : RUN yarn --production --non-interactive --silent     && yarn build >/dev/null
--
  | ---> Running in 6021a7fe08e4
  |  
  | warning " > @babel/plugin-syntax-dynamic-import@7.0.0-beta.51" has incorrect peer dependency "@babel/core@>=7.0.0-beta.50 <7.0.0-rc.0".
  | warning " > @patternfly/react-core@1.11.1" has unmet peer dependency "prop-types@^15.6.1".
  | warning "@patternfly/react-core > @patternfly/react-icons@2.0.0" has unmet peer dependency "prop-types@^15.6.1".
  | warning " > @patternfly/react-icons@2.4.0" has unmet peer dependency "prop-types@^15.6.1".
  | warning " > patternfly-react@2.13.1" has unmet peer dependency "prop-types@^15.6.0".
  | warning "patternfly-react > react-bootstrap-switch@15.5.3" has unmet peer dependency "prop-types@^15.5.0".
  | warning "patternfly-react > react-c3js@0.1.20" has unmet peer dependency "prop-types@^15.5.0".
  | warning "patternfly-react > reactabular-table@8.14.0" has unmet peer dependency "lodash@>= 4.0.0 < 5.0.0".
  | warning "patternfly-react > sortabular@1.6.0" has unmet peer dependency "lodash@>= 4.0.0 < 5.0.0".
  | warning "patternfly-react > table-resolver@3.3.0" has unmet peer dependency "lodash@>= 3.0.0 < 5.0.0".
  | warning "patternfly-react > table-resolver@3.3.0" has incorrect peer dependency "redux@>= 3.0.0 < 4.0.0".
  | error Command failed with exit code 2.
  | error Command failed with exit code 1.
  | Removing intermediate container 6021a7fe08e4
  | error: build error: The command '/bin/sh -c yarn --production --non-interactive --silent     && yarn build >/dev/null' returned a non-zero code: 1
```

**Expected behavior**
Build succeeds

Koku UI Commit: 9919087

--- a/package.json
+++ b/package.json
@@ -74,6 +74,7 @@
     "i18next-json-sync": "^2.2.0",
     "i18next-xhr-backend": "^1.5.1",
     "jest": "^22.4.3",
+    "jest-emotion": "^9.2.7",
     "mini-css-extract-plugin": "^0.4.0",
     "patternfly-react": "^2.13.1",
     "qs": "^6.5.2",
@@ -105,7 +106,6 @@
   },
   "devDependencies": {
     "husky": "^0.14.3",
-    "jest-emotion": "^9.2.7",
     "lint-staged": "^7.0.4"
   }
 }

